source/_integrations/honeywell.markdown

```diff
@@ -28,6 +30,14 @@ climate:
     password: YOUR_PASSWORD
 ```
 
+To add the sensors, add the following to the `sensor:` section of your `configuration.yaml` file:
+
+```yaml
+sensor:
+# Example configuration.yaml entry
+  - platform: honeywell
+```
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>